src/HotChocolate/Core/src/Types.CursorPagination/Extensions/UsePagingAttribute.cs

```diff
@@ -63,6 +65,24 @@ public bool IncludeTotalCount
             set => _includeTotalCount = value;
         }
 
+        /// <summary>
+        /// Allow forward paging using <c>first</c> and <c>after</c>
+        /// </summary>
+        public bool Forward
```

Comment:
Shouldn^t this be called AllowForwardNavigation? Or something in this direction?